generators/service-push/templates/python/Pipfile.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Shouldn't this have `requests` in it?

Reply:
Adding to this, are we sure that all of the `requirements.txt` files match exactly those of the `Pipfile.json`? I only recognized the inconsistency because I worked with service-push previously.